scripts/gulpfiles/config.js

```diff
@@ -38,6 +38,10 @@ exports.TYPINGS_BUILD_DIR = path.join(exports.BUILD_DIR, 'declarations');
 // Matches the value in tsconfig.json: outDir
 exports.TSC_OUTPUT_DIR = path.join(exports.BUILD_DIR, 'src');
 
+// Directory where typescript compiler can be found for `./tests/typescript`
+// files.
```

Comment:
This description seems confusing to me.  How about:
```suggestion
// Directory for files generated by compiling test code.
```

Reply:
done